testapps/RazorToolingTestApp/project.json

```diff
@@ -0,0 +1,48 @@
+﻿{
+  "dependencies": {
+    "Microsoft.AspNetCore.Diagnostics": "1.1.0-*",
+    "Microsoft.AspNetCore.Mvc": "1.1.0-*",
+    "Microsoft.AspNetCore.Razor.Design": {
+      "type": "build",
+      "version": "1.0.0-*",
+      "target": "package"
+    },
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.1.0-*",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0-*",
+    "Microsoft.Extensions.Logging.Console": "1.1.0-*",
+    "RazorToolingTestApp.Library": "1.0.0-*"
+  },
+
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*",
+    "Microsoft.AspNetCore.Razor.Tools": "1.0.0-*"
+  },
+
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "version": "1.0.0-*",
+          "type": "platform"
+        }
+      }
+    },
+    "net451": {},
+    "net46": {},
```

Comment:
We will probably need to drop...or skip on Linux. I'm not sure we have a way to target 4.6.2 in our current linux build system.

Can't wait for https://github.com/dotnet/corefx/issues/9735...it would make solving this trivial.